lib/actions/api.js

```diff
@@ -188,50 +206,7 @@ export function routingQuery(searchId = null, updateSearchInReducer = false) {
           dispatch(updateOtpUrlParams(state, searchId))
         }
 
-        // Also fetch a non-realtime route.
-        //
-        // FIXME: The statement below may no longer apply with future work
-        // involving realtime info embedded in the OTP response.
-        // (That action records an entry again in the middleware.)
-        // For users who opted in to store trip request history,
-        // to avoid recording the same trip request twice in the middleware,
-        // only add the user Authorization token to the request
-        // when querying the non-realtime route.
-        //
-        // The advantage of using non-realtime route is that the middleware will be able to
-        // record and provide the theoretical itinerary summary without having to query OTP again.
-        // FIXME: Interestingly, and this could be from a side effect elsewhere,
-        // when a logged-in user refreshes the page, the trip request in the URL is not recorded again
-        // (state.user stays unpopulated until after this function is called).
-        //
-        const { user } = state
-        const storeTripHistory =
-          user && user.loggedInUser && user.loggedInUser.storeTripHistory
-
-        const nonRealtimeFetch = fetch(
-          constructRoutingQuery(state, true, {
-            batchId: searchId,
-            ...injectedParams
-          }),
-          getOtpFetchOptions(state, storeTripHistory)
-        )
-          .then(getJsonAndCheckResponse)
-          .then((json) => {
-            // FIXME: This is only performed when ignoring realtimeupdates currently, just
-            // to ensure it is not repeated twice.
-            // FIXME: We should check that the mode combination actually has
-            // realtime (or maybe this is set in the config file) to determine
-            // whether this extra query to OTP is needed.
-            return dispatch(
-              nonRealtimeRoutingResponse({ response: json, searchId })
```

Comment:
Delete in the corresponding files:
- this action (this file is the only place it is used)
- `NON_REALTIME_ROUTING_RESPONSE` from the reducer
- instances of `activeSearch.nonRealtimeResponse`
- instances of `state.otp.useRealtime`
- Other vars and code that gets unused from the above.

Reply:
Would this result in a breaking change commit?

Reply:
See comment above -- I kept these thinking that we would re-use them when we refactor the monitored trips component. I'll add some comments to all these explaining why they're there. 

Something for the backlog though would be to clear out a lot of these reducers. There's plenty of unused ones floating around...

Reply:
I've gone ahead and removed `useRealtime` as it appeared to be a setting that wasn't used anywhere. That led me to have to add typescript in some places and also make a small refactor to the realtime status label. No breaking change though as `useRealtime` wasn't even in the example-config and hasn't been used in years.

Reply:
Using `useRealtime` prior to this commit wouldn't have done anything anyway